tools/docker/Dockerfile

```diff
@@ -36,3 +36,6 @@ RUN pip install .
 
 WORKDIR /src
 RUN rm -rf /src/packet-queue
+
+RUN groupadd whiterabbit
+RUN echo "%whiterabbit  ALL=(root) NOPASSWD: "`which impaired_network_server` >> /etc/sudoers.d/whiterabbit
```

Comment:
There doesn't seem to be any consistent rule about this, even in the default sudoers file; I changed to one space, but as far as I can tell, double spaces, tabs, etc. are all acceptable.